### ISSUE TITLE
[FEATURE] PSDefaultParameterValues are not respected by Connect-PnPOnline

### ISSUE THREAD

**Issue by JustinGrote** [pnp/powershell#52]
### Expected behavior 
PSDefaultParameter settings will remove the need to specify them manually each time.
### Actual behavior 
![image](https://user-images.githubusercontent.com/15258962/101948991-caa01c80-3ba7-11eb-9871-7cccbedf8a4b.png)

### Workaround
Splatting (sort of)

### What is the version of the Cmdlet module you are running?
0.3.6-nightly

### Which operating system/environment are you running PnP PowerShell on?
- [x] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by JustinGrote** [comment]
Probably because the cmdlets are imported as a nested module? Could be fixed by putting the nested module logic in a .psm1 file
https://github.com/PowerShell/PowerShell/issues/3206

**Comment by erwinvanhunen** [comment]
actually in the PSD1 you see that we set the prefix of the cmdlets to PnP, e.g. the cmdlet itself is actually called Connect-Online, but when loading the module we insert PnP, making the Cmdlet called Connect-PnPOnline.  This seems to work nicely, however, in the case of $PSDefaultParameterValues you have to address the cmdlet with it's original name: Connect-Online. If you do:

```powershell
$PSDefaultParameterValues=@{"Connect-Online:Url"="https://yoururl.sharepoint.com"}
Connect-PnPOnline
```

We'll discuss with the team if we should remove the prefix setting in the PSD1 and rename all cmdlets to hardcoded include the PnP prefix.

**Comment by erwinvanhunen** [comment]
We changed the internal code of PnP PowerShell to have hardcoded PnP prefixes. This will be available in the upcoming 0.3.7-nightly and after installing that one $PSDefaultParameterValues should work as expected.

**Comment by JustinGrote** [comment]
@erwinvanhunen interesting, I don't think this happens with script cmdlets and using prefix. I'll have to investigate. This probably should be opened as a separate issue in the Powershell repository though, as it is obviously undesired behavior :).

Thanks!
